src/integrationTest/java/uk/gov/hmcts/darts/ws/ContextRegistryDocumentumToJwtServiceSharedTokenTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.darts.ws;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.darts.cache.token.config.CacheProperties;
+import uk.gov.hmcts.darts.config.OauthTokenGenerator;
+import uk.gov.hmcts.darts.utils.client.ctxt.ContextRegistryClient;
+import uk.gov.hmcts.darts.utils.client.ctxt.ContextRegistryClientProvider;
+
+@ActiveProfiles("int-test-documentum-jwt-token-shared")
+class ContextRegistryDocumentumToJwtServiceSharedTokenTest extends ContextRegistryParent {
+    @MockBean
+    private OauthTokenGenerator generator;
+
+    @Autowired
+    private CacheProperties properties;
+
+    @Autowired
+    private RedisTemplate<String, Object> restTemplate;
```

Comment:
Should be renamed to be `redisTemplate`?

Reply:
Done. Actually wasnt being used so I have removed.